pkg/updater/updater.go

```diff
@@ -88,22 +83,28 @@ func NeedsUpdate(ctx context.Context, log logrus.FieldLogger, repo, version stri
 		log.WithError(err).Warn("failed to get user's home directory")
 		return false
 	}
+	cacheDir := filepath.Join(homedir, ".outreach", ".cache", ".updater")
```

Comment:
Should this follow the same app/repo pattern config does? (Consistency, feel free to ignore)

Reply:
It does, but it uses repo.yaml instead of a dedicated directory